### DISCUSSION TITLE
Error: "This project requires the 'AirSim' plugin..."

### DISCUSSION THREAD

**Discussion by jueunkwon** [microsoft/AirSim#3497] [Support Q&A] [answered]
Hello!

I was following directions for "custom unreal environment" from https://microsoft.github.io/AirSim/unreal_custenv/ to set up AirSim on Unreal Engine on Windows. I got an error in step 3 when I re-opened LandscapeMountains.sln in Visual Studio and clicked F5 to run. The error said "This project requires the 'AirSim' plugin, which could not be found. Would you like to disable it and continue", although I did have the AirSim plugin in the same folder.

Thank you in advance :)

**Comment by rajat2004** [comment, marked as answer]
That's pretty strange, maybe try repeating the steps from the beginning. Also, make sure to run `build.cmd` form root AirSim, and maybe even `update_from_git.bat` from `Unreal/Environments/Blocks` before copying the `Unreal/Plugins` folder if the AirSim installation is fresh. 

Also, if you find some instructions are missing or needs to be updated, then do open a PR or maybe comment on https://github.com/microsoft/AirSim/pull/3514 where I've made some small improvements

**Reply by jueunkwon** [reply]
Hello, thank you so much! I repeated the tutorial steps after updating and the error has been resolved.

However, whenever I open the  LandscapeMountains.uproject, I still get an error that says:
Default Property warnings and errors: 
Error: CDO Constructor (PIPCamera): Failed to find Material’/AirSim/HUDAssets/CameraDistortion.CameraDistortion’

I saw the post https://github.com/microsoft/AirSim/issues/3238 that said I should upgrade to UE 4.25 but Landscape Mountains is only supported until UE 4.24. Do you have any suggestions to fix this? Thank you!

**Reply by rajat2004** [reply]
You could try the changes mentioned in https://github.com/microsoft/AirSim/issues/3238#issuecomment-797769376, but I haven't myself tested it. As to the effects, if you don't use the distortion APIs, then it should be okay

**Reply by xxEoD2242** [reply]
Just to let you know, you can convert Landscape Mountains by installing UE 4.25 and just starting the project. It will automatically compile

**Comment by iibuan** [comment]
Hello, I encountered an error kind of similar to this. The error message is `This project requires the 'AirSim' plugin, which has a missing dependency on the 'PhysXVehicles' plugin.`

I tried to do fresh build of AirSim several times and I keep encountering this error.

**Reply by RISHIKBHIMAVARAPU** [reply]
even I am encountering the same 
did you find the solution for it ?

**Reply by iibuan** [reply]
Yes, I encountered this error because I use UE5. I installed UE4, and did a new build of AirSim. Now it works.

**Reply by WHaLe2400** [reply]
Is there any solutions that can make the airsim work in UE5 (T^T), I'v got so many work in UE5, and I don't want to restart ALL these to switch to UE4
